options.json

```diff
@@ -0,0 +1,28 @@
+
+{
+  "type": "object",
+  "properties": {
+    "base": {
+      "type": "number"
+    },
+    "attrs": {
+      "type": "object"
+    },
+    "insertAt": {
+      "type": "string"
+    },
+    "insertInto": {
+      "type": "string"
+    },
+    "singleton": {
+      "type": "boolean"
+    },
+    "sourceMap": {
+      "type": "boolean"
+    },
+    "convertToAbsoluteUrls": {
+      "type": "boolean"
+    }
```

Comment:
For some reason it doesn't get displayed in the `OptionsValidationError` @schema-utils, but agreed this is mandatory to some point, but I need to fix it in `schema-utils` first 😛